### PR TITLE
[DOCS] Updated URLs to reflect test/production

### PR DESCRIPTION
**Pull request by MalcolmSmithUK**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,13 +82,13 @@ Then include this script in your page code.
...
 ```html
-<script src="http://rci-prod-euwest1-sdk.s3.eu-west-1.amazonaws.com/latest/rci.min.js"></script>
+<script src="https://sdk.real-user-data.eggplant.cloud/latest/rci.min.js"></script>
 ```
 
 Or if you want to stick with a certain version you can bind to a specific version by using the following code:
 
 ```html
-<script src="http://rci-prod-euwest1-sdk.s3.eu-west-1.amazonaws.com/v1.0.8/rci.min.js"></script>
+<script src="https://sdk.real-user-data.eggplant.cloud/v1.0.8/rci.min.js"></script>
```
Comment (yghern) [inline README.md:91]: [DOD] we should also update example html

**Review by yghern — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MalcolmSmithUK) [reply README.md:91]: OK - Done

**Comment by robertdumitrescu** [comment]
Looks good.

**Review by robertdumitrescu — APPROVED** [review]
Looks good.

**Review by yghern — COMMENTED** [review]
Looks good

**Review by yghern — APPROVED** [review]

**Pull request merged**